.github/workflows/deploy.yml

```diff
@@ -118,21 +118,21 @@ jobs:
           KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
           GITHUB_RUN_NUMBER: ${{ github.run_number }}
         with:
-          lane: 'release_gps'
-          subdirectory: 'android'
-          bundle-install-path: 'vendor/bundle'
+          lane: "release_gps"
+          subdirectory: "android"
+          bundle-install-path: "vendor/bundle"
 
       - name: Upload to Google Play beta track
         uses: maierj/fastlane-action@v1.4.0
         with:
-          lane: 'play_store_gps'
-          subdirectory: 'android'
-          bundle-install-path: 'vendor/bundle'
+          lane: "play_store_gps"
+          subdirectory: "android"
+          bundle-install-path: "vendor/bundle"
 
   deploy_ios_gps:
     # gps-release branch only
     if: github.event_name == 'push' && github.ref == 'refs/heads/gps-release'
-    runs-on: macOS-latest
+    runs-on: macos-latest
```

Comment:
https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources 
this has been wrong, but github has been interpreting it correctly 👀